wrappers/vcx-napi-rs/index.d.ts

```diff
@@ -101,6 +120,7 @@ export function outOfBandReceiverCreate(msg: string): number
 export function outOfBandReceiverExtractMessage(handle: number): string
 export function outOfBandReceiverConnectionExists(handle: number, connHandles: Array<number>): Promise<number>
 export function outOfBandReceiverBuildConnection(handle: number): Promise<string>
+export function outOfBandReceiverBuildNonmediatedConnection(handle: number, serviceEndpoint: string, routingKeys: Array<string>): Promise<string>
```

Comment:
I understand this is mirroring the status quo, so no need to address it here, but: 

I think we should strive to invert this API - instead of calling something like this, the user should call just `Connection.createWithInvitation(oob)`, and in fact that should work even today I suppose. Perhaps this API is redundant to begin with.

Reply:
We are in absolute agreement on this point.